pages/blog/status-update-41-20.md

```diff
@@ -0,0 +1,75 @@
+- The tools section is now dedicated to AsyncAPI's official tools to clarify the AsyncAPI initiative is not just the spec. Just have a look how much detailed is, for example, the [generator view](/generator),
+- Don't worry, all the other tools are still there, under [the Community section](/docs/community/tooling)
+- The cherry on the cake is a much more appealing [Blog view](/blog)
+
+[Fran Méndez](https://twitter.com/fmvilas) did all this fantastic job. Well done, Bro! Sources of the website are here. Feel free to use it as a template for your website and let us know if you need help reusing it.
+
+## Hacktoberfest
+
+AsyncAPI Initiative joined Hacktoberfest. We decided to join the event at the end of August when nobody even thought this would happen. The first day of Hacktoberfest felt like a false start. Because of the enormous amount of SPAM pull requests, some open-source community people went too far in the #cancelculture trend. They opted on Twitter for the Hacktoberfest cancelation and DigitalOcean boycott. Luckily not everyone is haters and Hacktoberfest stays. Anyway, this is a topic for another post.
+
+Do we regret we engaged with event participants? Hell no! So far, yes, we got only 2 SPAM PRs, but we also got many issues addressed (25 in 7 days) that we listed for the event. 
+
+We had two primary goals to join the event:
+- Show the community that AsyncAPI Initiative is not just the spec but a lot of great tooling.
+- Help out people to take their first baby step in open-source contributions in a welcoming company. We not only prepared issues with different levels of difficulty. We also created a set of videos that explain how to contribute and what the issues are about. 
+
+Anyway. We share the summary of how Hacktoberfest ended up for us in the next status update. So far, so good though.
+
+Remember that throughout the entire October me and Fran, we'll be hosting office hours so anyone can join and ask for help or even do a pair programming session with us. Feel invited! More details in the blog post about the event. We will stream to our official media accounts:
+- https://www.twitch.tv/asyncapi
+- https://www.youtube.com/asyncapi
+- https://twitter.com/AsyncAPISpec 
+
+Look into [this](https://calendar.google.com/calendar/u/0?cid=dGJyYmZxNGRlNWJjbmd0OG9rdmV2NGxzdGtAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ) calendar for the schedule.
+
+## Apidays Live Hong Kong
+
+The AsyncAPI founder, [Fran Méndez](https://twitter.com/fmvilas), was invited to present at Apidays Live Hong Kong. It is a free event that we recommend you to join. Fran's talk is scheduled for tomorrow, 8.10 2:10PM (Hong Kong timezone). He'll talk about the AsyncAPI and the Future of API specs. To join the event, register [here](https://www.eventbrite.com/e/apidays-live-hong-kong-the-open-api-economy-finance-as-a-service-api-ecosystems-tickets-104511637120). 
```

Comment:
```suggestion
The AsyncAPI founder, [Fran Méndez](https://twitter.com/fmvilas), was invited to present at Apidays Live Hong Kong. It is a free event that we recommend you to join. Fran's talk is scheduled for tomorrow at 8.10 AM CEST (2:10 PM Hong Kong timezone). He'll talk about AsyncAPI and the Future of API specs. To join the event, register [here](https://www.eventbrite.com/e/apidays-live-hong-kong-the-open-api-economy-finance-as-a-service-api-ecosystems-tickets-104511637120). 
```

Reply:
I actuall meant 8th of October 😆 but yeah better to write 8 AM 😄

Reply:
LOL, was a coincidence. My talk is actually at 8:10 AM. BTW, you may want to use 8:10 instead of 8.10 in the end.